daemon/algod/api/server/v2/handlers.go

```diff
@@ -64,8 +64,8 @@ import (
 // MaxTealSourceBytes sets a size limit for TEAL source programs for requests
 // Max TEAL program size is currently 8k
 // but we allow for comments, spacing, and repeated consts
-// in the source TEAL, so we allow up to 200KB
-const MaxTealSourceBytes = 200_000
+// in the source TEAL, so we allow up to 1MB
+const MaxTealSourceBytes = 1_000_000
```

Comment:
If the previous hand-wavy argument that 200_000 was reasonable has caused problems, then there should be a number that is _correct_ in the sense that it can't cause those problems. But I don't see what those problems were, exactly, nor why 1MB is the number that is correct to solve them.

Reply:
my current teal code for clamm when i remove the comments from the code is 190kb .. when i keep the comments its 210kb.. i prefer not to code a feature to trim the comments from the higher level language because i like when the code is commented, and it is better for auditing process when teal code is commented

i believe 1MB is the better limit because this way devs will hit different limits and not the length of teal code program limit

Reply:
Could you tell me how many bytes your teal compiles to?  Since we are considering raising the potential size of a logicsig to 16kb, I'd like to have enough headroom to allow for program that compiles to something that big, along with a reasonable amount of comments.